python/ray/train/constants.py

```diff
@@ -84,3 +84,10 @@
 
 # Key for AIR Checkpoint metadata in TrainingResult metadata
 CHECKPOINT_METADATA_KEY = "checkpoint_metadata"
+
+# Key for AIR Checkpoint world rank in TrainingResult metadata
+CHECKPOINT_RANK_KEY = "checkpoint_rank"
+
+
+# Key for AIR Checkpoint that gets uploaded from distributed workers.
+CHECKPOINT_DISTRIBUTED_KEY = "distributed"
```

Comment:
Do we need this? Not used anywhere.

Reply:
yeah I plan to use this for sync down case.
it's not used yet.
basically, if you are trying to use a "distributed" checkpoint, we will double check to make sure the files are downloaded locally.